include/pdal/util/Bounds.hpp

```diff
@@ -189,6 +202,10 @@ class PDAL_DLL BOX3D : private BOX2D
     BOX3D()
        { clear(); }
 
+    BOX3D(const BOX2D& box)
```

Comment:
We got rid of this on purpose.  It might be OK with "explicit".  Why is it here?